Python/sysmodule.c

```diff
@@ -1994,6 +1994,42 @@ sys_getandroidapilevel_impl(PyObject *module)
 }
 #endif   /* ANDROID_API_LEVEL */
 
+/*[clinic input]
+sys._activate_perf_trampoline
+
+Activate the perf profiler trampoline.
+[clinic start generated code]*/
+
+static PyObject *
+sys__activate_perf_trampoline_impl(PyObject *module)
```

Comment:
That's useful, thanks! One could abuse `SIGPROF` and install a signal handler that activates perf trampoline on `SIGPROF`...